### PR TITLE
Update website-quickstart.md

### PR DESCRIPTION
**Pull request by anilkugupta**
spelling correction for \inetpub\wwwroot\

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[427dfef](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/427dfefafc04dd589cb5d6ed0b7adf88d4b2dfaa)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[427dfef](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/427dfefafc04dd589cb5d6ed0b7adf88d4b2dfaa)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

File: dsc/quickstarts/website-quickstart.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ title:  Quickstart - Create a website with DSC
 
 This exercise walks through creating and applying a Desired State Configuration (DSC) configuration from start to finish.
 The example we'll use ensures that a server has the `Web-Server` (IIS) feature enabled,
-and that the content for a simple "Hello World" website is present in the `intepub\wwwroot` directory of that server.
```
Comment (anilkugupta) [inline dsc/quickstarts/website-quickstart.md]: typo correction

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[427dfef](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/427dfefafc04dd589cb5d6ed0b7adf88d4b2dfaa)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/quickstarts/website-quickstart.md](https://github.com/anilkugupta/PowerShell-Docs/blob/patch-1/dsc/quickstarts/website-quickstart.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/quickstarts/website-quickstart?branch=pr-en-us-3714) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C14%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902141247360062-3714%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oXxwzUrfDT6ByQD0y%2BwhHtfOJBhO1z9HMSAVjfXJqlc%3D&st=2019-02-14T12%3A55%3A48Z&se=2019-03-17T13%3A00%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9dcf91f](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/9dcf91f663327f50124e81d3835535d633dbe2db)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9dcf91f](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/9dcf91f663327f50124e81d3835535d633dbe2db)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9dcf91f](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/9dcf91f663327f50124e81d3835535d633dbe2db)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/quickstarts/website-quickstart.md](https://github.com/anilkugupta/PowerShell-Docs/blob/patch-1/dsc/quickstarts/website-quickstart.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/quickstarts/website-quickstart?branch=pr-en-us-3714) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C14%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902141555284143-3714%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FDZLkk35j8Snd6BDjoUVj3fvmQp3QUHhXQ%2BZhOhxuUo%3D&st=2019-02-14T16%3A04%3A07Z&se=2019-03-17T16%3A09%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[e689a9b](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/e689a9bc75984b7bbc174285ddf289ceb8bd5607)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[e689a9b](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/e689a9bc75984b7bbc174285ddf289ceb8bd5607)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 48 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[e689a9b](https://github.com/anilkugupta/PowerShell-Docs/pull/3714/commits/e689a9bc75984b7bbc174285ddf289ceb8bd5607)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/quickstarts/website-quickstart.md](https://github.com/anilkugupta/PowerShell-Docs/blob/patch-1/dsc/quickstarts/website-quickstart.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/dsc/quickstarts/website-quickstart?branch=pr-en-us-3714) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C14%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902141641549440-3714%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BGdP4Z9sgzNaBlrv0ObV%2B6lr2YMk%2B0o17qv8TX8qOxs%3D&st=2019-02-14T16%3A50%3A24Z&se=2019-03-17T16%3A55%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**